### PR TITLE
round tables guides 5-7

### PR DESCRIPTION
**Pull request by beccaelenzil**
Proofreading and any feedback on the content of the questions is much appreciated. I also recognize we will have more information after a couple sessions to update the content of the questions to be most useful for the guided review.

### CODE REVIEW

File: roundtable-guides/05-loops.md [hunk 1]
```diff
@@ -1,21 +1,64 @@
...
 
 ## `for` loops
-range
-string (for each)
+1. In the `for` loop below
+    * How many times does the block execute?
+    * What values does `i` take on each time through the loop?
+
+    ```python
+    for i in range(2,5):
```
Comment (kaidamasaki) [inline roundtable-guides/05-loops.md:17]:
```suggestion
    for i in range(2, 5):
```

File: roundtable-guides/05-loops.md [hunk 2]
```diff
@@ -1,21 +1,64 @@
...
+1. What is the fundamental untility of using loops?
+1. What is the difference between a `for` loop and a `while` loop?
+    * When would you use one vs. the other?
 
 ## `for` loops
-range
-string (for each)
+1. In the `for` loop below
+    * How many times does the block execute?
+    * What values does `i` take on each time through the loop?
```
Comment (kaidamasaki) [inline roundtable-guides/05-loops.md:14]:
```suggestion
    * What value does `i` take on the first time through the loop?
```

File: roundtable-guides/05-loops.md [hunk 3]
```diff
@@ -1,21 +1,64 @@
...
+
+    ```python
+    for i in range(2,5):
+        print(i)
+    ```
+1. In the `for` loop below
+    * What values does `i` take on each time through the loop?
```
Comment (kaidamasaki) [inline roundtable-guides/05-loops.md:21]:
```suggestion
    * What value does `i` take on each time through the loop?
```

File: roundtable-guides/05-loops.md [hunk 4]
```diff
@@ -1,21 +1,64 @@
...
 
 ## Practice Problems
-reverse string
-totaling even numbers
+1. Let's share our code for `reverse_string` and `total_even_numbers`.
+1. What questions do you have about the practice problems?
 
 ## Rock, Paper, Scissors - v5
+1. Let's share our code for *Rock, Paper, Scissors - v3*
+1. What question do you have about *Rock, Paper, Scissors - v3*?
```
Comment (kaidamasaki) [inline roundtable-guides/05-loops.md:51]:
```suggestion
1. What questions do you have about *Rock, Paper, Scissors - v3*?
```

File: roundtable-guides/06-lists.md [hunk 5]
```diff
@@ -1,17 +1,57 @@
...
 
-Today's round table will focus on reviewing [lesson on lists](XXX).
+Today's round table will focus on reviewing [lesson on lists](https://colab.research.google.com/drive/1TK9Enhh0mITZ1649l-r4_gzeg2B3eRRu?usp=sharing).
 
 ## Creating Lists
+1. What are two different ways to initialize an empty list?
+    * Why would we use one way vs. the other?
+1. What are two different ways to initialize a list with data?
+    * Why would we use one way vs. the other?
+1. A single list in python can inlcude elements of multiple data types.
```
Comment (kaidamasaki) [inline roundtable-guides/06-lists.md:10]:
```suggestion
1. A single list in Python can include elements of multiple data types.
```

File: roundtable-guides/06-lists.md [hunk 6]
```diff
@@ -1,17 +1,57 @@
...
+    * What should we consider when combining different data types in a single list?
```
Comment (kaidamasaki) [inline roundtable-guides/06-lists.md:11]:
```suggestion
    * What should we consider when storing different data types in a single list?
```

File: roundtable-guides/06-lists.md [hunk 7]
```diff
@@ -1,17 +1,57 @@
...
+    * What questions do you have about nesting a conditional control structure in a for-each loop?
+
+## Project: Account Generator
+1. What are the benefits of creating a list of dictionaries to store account information?
+1. Let's share our code for *Account Generator*
+1. What question do you have about *Account Generator*?
+
+## Project: Rock, Paper, Scissors - v4
+1. Let's share our code for *Rock, Paper, Scissors - v4*
+1. What question do you have about *Rock, Paper, Scissors - v4*?
```
Comment (kaidamasaki) [inline roundtable-guides/06-lists.md:57]:
```suggestion
1. What questions do you have about *Rock, Paper, Scissors - v4*?
```

File: roundtable-guides/05-loops.md [hunk 8]
```diff
@@ -1,21 +1,64 @@
...
+    * What extra column is included in a loop table for a `while` loop compared to a `for` loop?
+1. What other strategies have you used for debugging loops?
 
 ## Practice Problems
-reverse string
-totaling even numbers
+1. Let's share our code for `reverse_string` and `total_even_numbers`.
+1. What questions do you have about the practice problems?
 
 ## Rock, Paper, Scissors - v5
```
Comment (kaidamasaki) [inline roundtable-guides/05-loops.md:49]: The questions refer to "v3" not "v5".

```suggestion
## Rock, Paper, Scissors - v3
```

File: roundtable-guides/07-dictionaries.md [hunk 9]
```diff
@@ -1,19 +1,52 @@
...
 
-Today's round table will focus on reviewing [lesson on dictionaries](XXX).
+Today's round table will focus on reviewing [lesson on dictionaries](https://colab.research.google.com/drive/1AmKeKvSJnNacUUIU9OLSInVohWJrPLkF?usp=sharing).
 
 ## Creating Dictionaries
+1. Why is it useful to organize data in a dictionary?
+1. Describe two ways to create an empty dictionary.
 
 ## Accessing Values
+1. Describe two ways to access the value stored in a particular key?
```
Comment (kaidamasaki) [inline roundtable-guides/07-dictionaries.md:10]:
```suggestion
1. Describe two ways to access the value associated with a particular key?
```

File: roundtable-guides/07-dictionaries.md [hunk 10]
```diff
@@ -1,19 +1,52 @@
...
+1. What happens if you replicate keys in a dictionary?
```
Comment (kaidamasaki) [inline roundtable-guides/07-dictionaries.md:11]:
```suggestion
1. What happens if you duplicate keys in a dictionary?
```

File: roundtable-guides/07-dictionaries.md [hunk 11]
```diff
@@ -1,19 +1,52 @@
...
+    `numbers = {'a': 1, 'a': 2}`
+    * What does numbers['a'] return? 
+1. Only immutable types can be used as keys. What does it mean for an object to be immutable?
+1. Can you use numerical indices to access values from a dictionary. Why or why not?
```
Comment (kaidamasaki) [inline roundtable-guides/07-dictionaries.md:15]: Just to make the questions stand out more:

```suggestion
1. Only immutable types can be used as keys. 
  * What does it mean for an object to be immutable?
1. Can you use numerical indices to access values inside a dictionary? 
  * Why or why not?
```

File: roundtable-guides/07-dictionaries.md [hunk 12]
```diff
@@ -1,19 +1,52 @@
...
+    * What questions do you have?
+1. Let's look at the second to last line in *Problem 6*
+    `if(not(oatmeal_raisin["gluten_free"]) or not(oatmeal_raisin["vegan"]))`
+    * How could the second to last line be refactored without using the `not` operator?
+    * How could it be refactored using a chained conditional (`if/elif/...`)
+
+## Project: Account Generator - v2
+
+1. Let's share our code for *Account Generator - v2*
+1. What question do you have about *Account Generator - v2*?
```
Comment (kaidamasaki) [inline roundtable-guides/07-dictionaries.md:47]:
```suggestion
1. What questions do you have about *Account Generator - v2*?
```

File: roundtable-guides/06-lists.md [hunk 13]
```diff
@@ -1,17 +1,57 @@
...
+    * What are two ways to access the last element `'e'`.
+    * What does `letters[1]` return?
+    * What does `letters[5]` return?
 
 ## List Methods
+1. The Ada Build lesson introduces the `len` and `append` methods.
+    * What other methods have you explored?
 
 ## Lists and Loops
+1. The Ada Build lessons shows two different ways of looping over a list:
```
Comment (spitsfire) [inline roundtable-guides/06-lists.md:24]:
```suggestion
1. The Ada Build lessons show two different ways of looping over a list:
```

File: roundtable-guides/06-lists.md [hunk 14]
```diff
@@ -1,17 +1,57 @@
...
+    * What are some of the benefits and drawbacks of each of these techniques?
 
 ## Practice Problems
-
-## Account Generator
-
-## Rock, Paper, Scissors - v4
\ No newline at end of file
+1. What questions do you have about the practice problems?
+1. Let's Look at Problem 2.5
```
Comment (spitsfire) [inline roundtable-guides/06-lists.md:39]:
```suggestion
1. Let's look at problem 2.5
```

File: roundtable-guides/06-lists.md [hunk 15]
```diff
@@ -1,17 +1,57 @@
...
+            print("Special case!")
+        else:
+            print(f"Regular values like {value}")
+    ```
+    * What questions do you have about nesting a conditional control structure in a for-each loop?
+
+## Project: Account Generator
+1. What are the benefits of creating a list of dictionaries to store account information?
+1. Let's share our code for *Account Generator*
+1. What question do you have about *Account Generator*?
```
Comment (spitsfire) [inline roundtable-guides/06-lists.md:53]:
```suggestion
1. What questions do you have about *Account Generator*?
```

File: roundtable-guides/06-lists.md [hunk 16]
```diff
@@ -1,17 +1,57 @@
...
+
+## Project: Rock, Paper, Scissors - v4
+1. Let's share our code for *Rock, Paper, Scissors - v4*
+1. What question do you have about *Rock, Paper, Scissors - v4*?
```
Comment (spitsfire) [inline roundtable-guides/06-lists.md:57]:
```suggestion
1. What questions do you have about *Rock, Paper, Scissors - v4*?
```

**Review by spitsfire — CHANGES_REQUESTED** [review]

File: roundtable-guides/05-loops.md [hunk 17]
```diff
@@ -1,21 +1,64 @@
 # Lesson 05: Loops and Iteration
 
-Today's round table will focus on reviewing the [lesson on loops](XXX).
+Today's round table will focus on reviewing the [lesson on loops](https://colab.research.google.com/drive/1m9h053kS6bjAeiHnEHIP39fqbyOO7glc?usp=sharing).
+
+## Loops
+
+1. What is the fundamental untility of using loops?
```
Comment (smarchante1) [inline roundtable-guides/05-loops.md:7]:
```suggestion
1. What is the fundamental utility of using loops?
```

**Review by smarchante1 — CHANGES_REQUESTED** [review]

**Review by spitsfire — APPROVED** [review]

**Pull request merged**